build.sbt

```
@@ -86,6 +84,8 @@ lazy val root = project
     ),
   )
   .aggregate(utils)
+  .aggregate(`products-shared-kernel`)
+  .aggregate(stocking)
```

Put those in a single comma separated line, it's what Nicolas did in his PR 
```suggestion
  .aggregate(utils, `products-shared-kernel`, stocking)
```